Code/GraphMol/MolDraw2D/DrawMol.cpp

```diff
@@ -1660,11 +1660,20 @@ void DrawMol::makeDoubleBondLines(
     // in, for example, an aldehyde, such as in catch_tests.cpp's
     // testGithub_5269_2.svg, might be asymmetrically shorter, so we don't
     // want to change colour at halfway
-    if (bond->getEndAtom()->getDegree() == 1 && !(cols.first == cols.second) &&
-        fabs((l1s - l1f).lengthSq() - (l2s - l2f).lengthSq()) > 0.01) {
-      double midlen = (l1s - l1f).length() / 2.0;
-      Point2D lineDir = l2s.directionVector(l2f);
-      Point2D notMid = l2s + lineDir * midlen;
+    auto l1 = (l1s - l1f).lengthSq();
+    auto l2 = (l2s - l2f).lengthSq();
+    if ((bond->getBeginAtom()->getDegree() == 1 ||
+         bond->getEndAtom()->getDegree() == 1) &&
+        !(cols.first == cols.second) && fabs(l1 - l2) > 0.01) {
+      double midlen = sqrt(l1) / 2.0;
```

Comment:
```suggestion
      double midlen = sqrt(l1) * 0.5;
```

Reply:
I always let the compiler sort out this sort of micro-optimisation.  Personally I think the intention is more immediately obvious with / 2.0 than * 0.5 but that might just be me.  I can change it if it's house style.

Reply:
More than a micro-optimization my suggestion was intended for consistency with the `* 0.5` that is used a few lines below (line 2799), but again, not a must.

Reply:
Ah, sorry, I missed that.  Consistency within a function is very desirable.  I will alter it.  If nothing else, that will force another run of the CI checks which will hopefully pass this time.

Reply:
At a quick scan there is pretty much a 50:50 mixture of / 2.0 and * 0.5 through this file.  I think I will leave this particular instance, but aim for more consistency in future.